src/core/load-fragment.c

```diff
@@ -1338,6 +1341,12 @@ int config_parse_exec_cpu_affinity(const char *unit,
         assert(rvalue);
         assert(data);
 
+        if (streq(rvalue, "NUMA")) {
+                c->numa_policy.extend_cpu_affinity = true;
```

Comment:
shouldn't this also clear out c->cpu_set?

Reply:
`c->cpu_set` is currently extended each time we parse new `CPUAffinity=` line. Hence special "numa" value works the same way. We just extend the mask instead of fully replacing it.